apps/predbat/predbat.py

```diff
@@ -498,7 +556,20 @@ def update_status(self, minutes_now, quiet=False):
 
     def adjust_reserve(self, reserve):
         """
-        Adjust the reserve target % in GivTCP
+        Adjust the output reserve target %
+
+        Inverter Class Parameters
+        =========================
+
+            None            
```

Comment:
Well, it's using self.rest_data, self.reserve_percent, self.reserve_max and references to base

Reply:
I did think about list what was being read but I'm mainly concerned that I can change the outputs of each function as required by a different inverter. I'm assuming that the inputs will be handled by the Predbat class.